wiki/Tournaments/Official_support/en.md

```diff
@@ -86,7 +92,15 @@ Once the tournament has concluded, the tournament organisers will need to submit
 
 Tournament organisers are expected to ensure that their tournaments run smoothly and with minimal disruption where possible.
 
-Users under an active tournament ban are expected to disclose their tournament ban status to any officially supported tournament they intend to help with. They may be enlisted as [helpers](#what-is-considered-staff) at the host's discretion, but we encourage careful consideration of such choices as said individuals have already infringed upon the rules once (or more). They may not act as [staff](#what-is-considered-staff) UNLESS the host requests an exemption for them when sending in their initial request for support. The account support team will then assess these on a case-by-case basis with the user's history in mind.
+With regards to staff roles, the following rules apply:
+
+- Streamers, commentators, and graphic designers may play in the tournament.
+- Eliminated players may be enlisted as referees, mappool playtesters, and/or replay creators.
+- Staff with any role not listed here, may not play in a badged tournament.
```

Comment:
It seems kind of unintuitive to blanket disallow all other roles from playing here if the staff scope exceeds the usual framework. I don't necessarily disagree because ambiguity will lead to loopholes, but maybe some clarity and/or leniency is in order.

Examples of roles that would technically be unallowed by this wording are:
* Statisticians, spreadsheet managers/editors, developers, website/wiki managers/editors, audio designers, Discord/Twitch moderators... I'd say those are examples of roles that could be listed but still be allowed to play, and I believe people can come up with more of them as the scene grows
* Music composers/producers: that one may fall under the pooling umbrella
* Hosts/coordinators: well yeah, obviously not playing, but just to cover our bases
* Consultant: this is an example of a vague title and I understand and agree with the point of blanket disallowing to avoid ambiguity for any title really: maybe an extra word/sentence is deserved to explain how to deal with those extra roles (e.g. make them fit into other roles).
* Staffing related to IRL/LAN events (e.g. logistics): I don't think they fall under "tournament staff" strictly speaking, so maybe they shouldn't be listed under the scope of tournament badging necessarily

Reply:
Email has been added to account for any edge-cases or questions that may arise when considering vague titles or positions that are unique to certain tournaments. This should solve the issues you had.

To clarify, this section is purely a rewrite of the rules that were in place previously and does not have any changes to which roles can staff and play in tournaments.